Misc/NEWS.d/next/Windows/2024-04-24-05-16-32.gh-issue-118209.Ryyzlz.rst

```diff
@@ -0,0 +1,12 @@
+Fix crashes in
+
+* :func:`mmap.move`
+* :func:`mmap.read`
+* :func:`mmap.read_byte`
+* :func:`mmap.readline`
+* :func:`mmap.write`
+* :func:`mmap.write_byte`
+* :func:`mmap.__getitem__`
+* :func:`mmap.__setitem__`
+
+on Windows when the underlying file operation raises errors or an access violation occurs.
```

Comment:
```suggestion
Avoid crashing in :mod:`mmap` on Windows when the mapped memory is inaccessible
due to file system errors or access violations.
```

Let's just keep this simple, as it's going to be in a very long list of changes.

However, in `Doc/whatsnew/3.13.rst` you'll find a section for "Improved modules" that contains "mmap" (or can be added), and there you can add something like this:

```
* :class:`mmap.mmap` is now protected from crashing on Windows when the mapped memory
  is inaccessible due to file system errors or access violations.
  (Contributed by <your name here> in :gh:`118209`)
```

Reply:
It's probably worth adding a note to `Doc/library/mmap.rst` saying that "most operations on Windows are protected from crashing due to file system errors or access violations, but may raise ``OSError`` if one of these occurs".

I don't think we need to necessarily point at each individual function.